src/cmd/singularity/cli/sign.go

```diff
@@ -17,26 +17,36 @@ import (
 
 func init() {
 	SignCmd.Flags().SetInterspersed(false)
+	SignCmd.Flags().StringVarP(&keyServerURL, "url", "u", defaultKeysServer, "specify the key server URL")
 	SingularityCmd.AddCommand(SignCmd)
 }
 
 // SignCmd singularity sign
 var SignCmd = &cobra.Command{
 	DisableFlagsInUseLine: true,
-	Args:   cobra.ExactArgs(1),
+	Args:   cobra.RangeArgs(1, 2),
```

Comment:
I think this should be `cobra.ExactArgs(1)`, as `sign` only takes one positional argument.